apps/mudita-center-e2e/src/specs/help/help-window-check.e2e.ts

```diff
@@ -5,23 +5,97 @@
 
 import NavigationTabs from "../../page-objects/tabs.page"
 import HelpPage from "../../page-objects/help.page"
+import HomePage from "../../page-objects/home.page"
+import HelpModalPage from "../../page-objects/help-modal.page"
+import NewsPage from "../../page-objects/news.page"
 
-describe("Check new window display in Help", () => {
+describe("Check Help window", () => {
   before(async () => {
-    // Waiting for device connected through USB
-    await browser.executeAsync((done) => {
-      setTimeout(done, 10000)
-    })
+    const notNowButton = await HomePage.notNowButton
+    await notNowButton.waitForDisplayed()
+    await notNowButton.click()
   })
-  it("Clicks help tab, switches to new window and checks if help topic is displayed in new window", async () => {
+
+  it("Open Help window", async () => {
     const helpTab = await NavigationTabs.helpTab
     await helpTab.waitForDisplayed({ timeout: 15000 })
     await helpTab.click()
 
     await browser.switchWindow("#/help")
+  })
+
+  it("Check contents of Mudita Help", async () => {
+    // Check window title
+    const helpTitle = await HelpPage.windowTitle
+    await helpTitle.waitForDisplayed({ timeout: 15000 })
+    await expect(helpTitle).toBeDisplayed()
+    await expect(helpTitle).toHaveText("Mudita Center Help")
+
+    // Check presence of the search engine
+    const searchIcon = await HelpPage.searchIcon
+    await searchIcon.waitForDisplayed()
+    await expect(searchIcon).toBeDisplayed()
+
+    const searchPlaceholder = await HelpPage.searchPlaceholder
+    await expect(searchPlaceholder).toHaveAttributeContaining("placeholder", "Search")
+
+    // Check presence of Contact support button
+    const contactSupportButton = await HelpPage.contactSupportButton
+    await contactSupportButton.waitForDisplayed()
+    await expect(contactSupportButton).toBeDisplayed()
+    await expect(contactSupportButton).toBeClickable()
+
+    const contactSupportTooltip = await HelpPage.contactSupportButtonTooltip
+    contactSupportButton.moveTo()
+    await expect(contactSupportTooltip).toBeDisplayed()
+    await expect(contactSupportTooltip).toHaveText("Contact support")
 
-    const helpTopic = await HelpPage.helpListElement
-    await helpTopic.waitForDisplayed({ timeout: 15000 })
+    // Check accordion
+    const helpTopic = await HelpPage.listElement
+    await helpTopic.waitForDisplayed()
     await expect(helpTopic).toBeDisplayed()
+    const no_of_articles = await $$('[data-testid="help-component-question"]').length
```

Comment:
```suggestion
    const noOfArticles = await $$('[data-testid="help-component-question"]').length
```
Consider renaming no_of_articles to noOfArticles to follow JavaScript naming conventions (camelCase).

Reply:
corrected